### PR TITLE
Option to force-update OMIM panel

### PR DESCRIPTION
**Pull request by northwestwitch**
Partially fixing #3399. I'm thinking to have the fix for update Panel app panel in another PR.

Right now it is not possible to update the OMIM panel before 24h from the last update. This PR allows force updating the panel, so a new version panel will be created when --force param is passed via the cli

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the paxa software installed on `hasta`:
 - Log in into hasta: `ssh <USER.NAME>@hasta.scilifelab.se`
 - Activate the staging environment with the command `us` (Use Stage)
 - Run the command `paxa` and follow the instructions. Make sure to specify scout-stage as the resource you request and the server cg-vm1 as server.
1. Log out from the hasta server.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>
1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Main branch:

image


This branch:

image

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -19,8 +19,9 @@ This PR marks a new Scout release. We apply semantic versioning. This is a major
 1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
 1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
 1. Make sure the branch is deployed: `systemctl --user status scout.target`
-</details>
 1. After testing is done, log out from `cg-vm1` and log in again in the `hasta` server, repeat the `hasta` and `paxa` procedure, which will release the allocated resource (scout-stage) to be used for testing by other users.
+</details>
```
Comment (dnil) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: 💯

File: scout/adapter/mongo/institute.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import logging
 from datetime import datetime
-from pprint import pprint as pp
```
Comment (dnil) [inline scout/adapter/mongo/institute.py:3]: 👍

File: scout/commands/update/omim.py [hunk 3]
```diff
@@ -29,8 +29,14 @@
     type=click.Path(exists=True),
     help="Path to file in omim mim2genes format",
 )
+@click.option(
```
Comment (dnil) [inline scout/commands/update/omim.py:32]: 👍

File: scout/adapter/mongo/panel.py [hunk 4]
```diff
@@ -77,10 +78,9 @@ def load_omim_panel(self, genemap2_lines, mim2gene_lines, institute=None):
 
         date_obj = get_date(date_string)
 
-        if existing_panel:
-            if existing_panel["date"] == date_obj:
-                LOG.warning("There is no new version of OMIM")
-                return
+        if existing_panel and existing_panel["date"] == date_obj and force is False:
```
Comment (dnil) [inline scout/adapter/mongo/panel.py:81]: My head hurts a little, but yes, agreed!

**Review by dnil — APPROVED** [review]
🥇

**Pull request merged**